### ISSUE TITLE
This is a duplicated project

### ISSUE THREAD

**Issue by alikhaleghi76** [pouyamaroufi/Persian-Linear-Date-Picker#4]
I'm opening this issue because you closed issue #3 before the problem get resolved. 
As I said this is an exact copy of my project at [this link](https://github.com/alikhaleghi76/flutter-datepicker).

I think it would be more professional to fork from main project, and send PR to fix code or add new changes.
If the developer announces that the main project is discontinued then you can publish your own version referring main project.

Any way please fork my project and merge your changes in that, or at least you should add a link to my project in descriptions of project at Github and pub.dev.

I hope it's not true but if you've used this project in your portfolio then not giving a link to main project is a bad practice and can have bad effects in your job seeking progress. 

Thank you.

**Comment by pouyamaroufi** [comment]
You are totally right. And in previous issue I told you that you are the one who can decide for package not me.If you want me to delete the Repo I will. It was just a solution for that time. 

I will delete this Repo ASAP from anywhere and Publish my Persian_Date_Pack that have nothing in common with your package.You can compare it with your package after release. 

sorry about things that happened and I apologise.
I didn't get any promotion or anything for releasing this package. I just wanted to help Iranians developers at those moments.This is why I release it on my website which other developers can reach it out easil.I thought you guys didn't support package anymore.

Please confirm this message for me and I'll delete Repo and anything related after that.

**Comment by alikhaleghi76** [comment]
There's no need to remove the package but I'm asking you to just refer to my package in the description of your project. 
Thank you.

**Comment by pouyamaroufi** [comment]
>  
> 
> There's no need to remove the package but I'm asking you to just refer to my package in the description of your project. 
> 
> Thank you.
> 
> 

I will

**Comment by alikhaleghi76** [comment]
> > There's no need to remove the package but I'm asking you to just refer to my package in the description of your project.
> > Thank you.
> 
> I will

I think 5 month is enough to change the readme of a project :)
Now my mind has changed and Im asking you to make the project private and make it discontinued on pub.dev
